src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(Color color, int side) {
+        super(Shape.SQUARE, color);
+        this.side = side;
+    }
+
+    public int getSide() {
+        return side;
+    }
+
+    public void setSide(int side) {
+        this.side = side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getShape() + ", area: " + getArea()
+                + " sq.units, side: " + side + " sq.units, color: " + getColor());
```

Comment:
`getColor()` return `Color` class and call method `toString()` in its. Use name() for getting String representation of enum constants. Fix this in all places